docs/source/guide/events.rst

```diff
@@ -675,7 +676,7 @@ about each event can be found in the corresponding sections below:
 
 
 `after-call`
-~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~
```

Comment:
Can we just standardize how long these separators are? Everything else looks fine.

Reply:
Per offline discussion it's difficult to standardize these because it must at least be as long as the length of the section header. If we chose an exact number it's always possible that we will have a section header that is longer than the number we choose. So it's probably best to make them the same length as the header.